DataFormats/BTauReco/src/classes_def.xml

```diff
@@ -302,7 +302,6 @@
   <class name="std::vector<Measurement1D>"/>
 
   <!-- RefToBase stuff -->
-  <class name="edm::RefToBase<reco::BaseTagInfo>" />
   <class name="edm::reftobase::BaseHolder<reco::BaseTagInfo>" />
```

Comment:
This PR was incomplete. All these `::reftobase::` dictionaries should be cleaned up as well. Could you do that?